src/pregh-changes.md

```diff
@@ -1,4 +1,12 @@
 ## 1.1.1
+=======
+1 Changelog
+
+1.2.0
+
+-   Added support for EEG data (merged BEP006)
+
+1.1.1
```

Comment:
Please don't change this file. The changelog is auto generated based on merged pull requests.

Reply:
+1